open_event/static/admin/js/scheduler.js

```diff
@@ -0,0 +1,236 @@
+/**
+ * Created by Niranjan on 23-May-16.
+ */
+(function ($) {
+    $(document).ready(function () {
+        /**
+         *  24px == 15 minutes
+         *  (Smallest unit of measurement is 15 minutes)
+         *
+         */
+
+        // TIME SETTINGS
+        var time = {
+            start: {
+                hours: 9,
+                minutes: 0
+            },
+            end: {
+                hours: 17,
+                minutes: 0
+            },
+            unit: {
+                minutes: 15,
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 14 space characters but found 16.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2618824881/issues/source?bid=3233864&fileBranchId=3313342#l23)